connector-model/src/main/resources/user-filter-definition-descriptor.xsd

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
WE don't want this one. Like for implementation, only connector definition should be used, even for filters